rockcraft.yaml

```diff
@@ -0,0 +1,75 @@
+name: spark
+summary: Spark ROCK
+description: Spark ROCK
+license: Apache-2.0
+
+version: "0.1"
+base: ubuntu:20.04
+
+entrypoint: ["/opt/entrypoint"]
+
+parts:
+  spark:
+    plugin: dump
+    source: https://downloads.apache.org/spark/spark-3.3.0/spark-3.3.0-bin-hadoop3.tgz
+    source-checksum: 1e8234d0c1d2ab4462d6b0dfe5b54f2851dcd883378e0ed756140e10adfb5be4123961b521140f580e364c239872ea5a9f813a20b73c69cb6d4e95da2575c29c
+    overlay-script: |
+      set -ex
+      sed -i 's/http:\/\/deb.\(.*\)/https:\/\/deb.\1/g' /etc/apt/sources.list
+      apt-get update
+      ln -s /lib /lib64
+      apt install -y bash tini libc6 libpam-modules krb5-user libnss3 procps
+      mkdir -p /opt/spark
+      mkdir -p /opt/spark/examples
+      mkdir -p /opt/spark/work-dir
+      touch /opt/spark/RELEASE
+      rm /bin/sh
+      ln -sv /bin/bash /bin/sh
+      echo "auth required pam_wheel.so use_uid" >> /etc/pam.d/su
+      chgrp root /etc/passwd && chmod ug+rw /etc/passwd
```

Comment:
I think this is basically a noop

Reply:
yes, /etc/passwd should be not changeable. we need to find a way to avoid this